features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/processor/RumContextDataHandler.kt

```diff
@@ -36,4 +42,9 @@ internal class RumContextDataHandler(
 
         return rumContextData
     }
+
+    companion object {
+        const val INVALID_RUM_CONTEXT_ERROR_MESSAGE = "SR RumContextDataHandler: Invalid RUM " +
```

Comment:
Maybe we could add the context in question to better understand what is not present and make the telemetry more actionable?